lmcache/v1/rpc_utils.py

```diff
@@ -95,11 +96,12 @@ def get_zmq_rpc_path_lmcache(
     import vllm.envs as envs
 
     base_url = envs.VLLM_RPC_BASE_PATH
+    pid = os.getpid()
```

Comment:
The  pid of each vllm process are all different, so how to communicate each other? @Foreverythin